src/Conda.jl

```diff
@@ -77,7 +77,11 @@ const conda = if Compat.Sys.iswindows()
     conda_bat = joinpath(p, "conda.bat")
     isfile(conda_bat) ? conda_bat : joinpath(p, "conda.exe")
 else
-    joinpath(bin_dir(ROOTENV), "conda")
+    if haskey(ENV, "CONDA_EXE")
```

Comment:
This whole `if` can be written as `get(ENV, "CONDA_EXE", joinpath(bin_dir(ROOTENV), "conda"))`.

Reply:
Thanks for the clean solution, I am still quite new to Julia -- TIL!

You are right, `CONDA_EXE` is not documented in the quite obsolete [user document](https://docs.conda.io/projects/conda/en/latest/index.html), albeit being mentioned in [release note](https://docs.conda.io/projects/conda/en/latest/release-notes.html?highlight=CONDA_EXE). 

However [conda/conda#7126](https://github.com/conda/conda/issues/7126) (with detailed [gitter chat record](https://gitter.im/conda/conda?at=5ac4e56dc574b1aa3e67a0d8)) might be quite evidential that `$CONDA_EXE` is considerably the de facto way to refer to conda executable after `conda 4.4+`...